runtimes/rocm-pytorch/ubi9-python-3.11/Pipfile

```diff
@@ -0,0 +1,58 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[[source]]
+url = "https://download.pytorch.org/whl/rocm6.0/"
```

Comment:
Should we want to have applied changes in #646 and #656 here already or we plan to perform these changes in the future? :thinking:

Reply:
Yep. I'm watching what needs to be incorporated into this PR. 
#646 is already in. #656 will be in soon.

Reply:
#656 done in e98ca27bf616397bcd319a5059b9705951d51852